packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -212,22 +217,21 @@ export class Cluster extends Resource implements ICluster {
       this._executeCommandConfiguration = props.executeCommandConfiguration;
     }
 
-    const cluster = new CfnCluster(this, 'Resource', {
```

Comment:
Are these expected changes? Could these be reverting the changes merged to main after the release branch was created?

Reply:
Yes, this change is expected